tasks/marketplacesvm-push-disk-images/README.md

```diff
@@ -10,4 +10,8 @@ It currently supports images in `raw` and `vhd` formats for `AWS` and `Azure` re
 | ----------------------- | -------------------------------------------------------------------------------------- | -------- | --------------- |
 | snapshotPath            | Path to the JSON string of the mapped snapshot spec in the data workspace.             | No       | -               |
 | cloudMarketplacesSecret | Env specific secret containing the marketplaces credentials.                           | No       | -               |
+| prePush                 | Whether perform a pre-push (true) or not (false). When true it will not publish PROD.  | No       | false           |
```

Comment:
It has default, so it is optional, right?

Reply:
Yes, it is! My mistake here, I'll adjust it